sites/header/_base.scss

```diff
@@ -0,0 +1,31 @@
+/*
+ * Site header
+ */
+
+.home #wb-bnr + hr {
+	border-top: 1px solid #ddd;
+	color: #284162;
+	margin-left: 0px;
+
+	.gcweb-menu button[aria-haspopup=true] {
+		background-color: #fff;
+		border-color: #fff;
+		color: #284162;
+
+		&:hover {
+			background-color: #444;
+			color: #fff;
+		}
+	}
+}
+
+#wb-bnr + hr {
+	border-top: 3px solid #38414d;
```

Comment:
This seem to duplicate this CSS instruction: https://github.com/wet-boew/GCWeb/pull/1903/files#diff-db183d6ace93ab416e893cbdbf8e5c42b0b66f862d26df75da7ad91c38fb5444R38
declared in gcweb-menu.scss

The style need to be defined in one or another but not duplicated.